scripts/run_autotyping.py

```diff
@@ -0,0 +1,38 @@
+"""
+Script to run ``autotyping``, to get around the fact that
+pre-commit puts ``args`` before the list of files, whereas
+``autotyping`` wants the files to come after, see
+https://github.com/pandas-dev/pandas/issues/48808#issuecomment-1259711679.
+"""
+from __future__ import annotations
+
+import argparse
+import subprocess
+import sys
+from typing import Sequence
+
+
+def main(argv: Sequence[str] | None = None) -> None:
+    parser = argparse.ArgumentParser()
+    parser.add_argument("paths", nargs="*")
+    args = parser.parse_args(argv)
+    output = subprocess.run(
+        [
+            "python",
+            "-m",
+            "libcst.tool",
+            "codemod",
+            "autotyping.AutotypeCommand",
+            *args.paths,
+            "--aggressive",
+        ],
+        capture_output=True,
+        text=True,
+    )
+    sys.stdout.write(output.stdout)
+    sys.stdout.write(output.stderr)
```

Comment:
Has this the same effect as removing `capture_output=True` and removing those two write statements?

Reply:
yup - turns out I'd always been overcomplicating using `subprocess.run`, thanks!